uint8 NO_CONSENT=0
uint8 DEPTH_ONLY=1
uint8 DEPTH_AND_RGB=2

uint8 result
